transforms/src/__testfixtures__/button-v0-to-v1.input.tsx

```diff
@@ -0,0 +1,42 @@
+import { Button, PrimaryButton, StealthButton } from '@fluentui/react-native';
+import { IFocusable } from '@fluentui-react-native/interactive-hooks';
+import { Stack } from '@fluentui-react-native/stack';
+import * as React from 'react';
+import { findNodeHandle } from 'react-native';
+import { SvgIconProps } from '@fluentui-react-native/icon';
+import TestSvg from '../test.svg';
+
+export const ButtonFocusTest_deprecated: React.FunctionComponent = () => {
+  const [state, setState] = React.useState({
+    focused: false,
+  });
+  const buttonRef = React.useRef<IFocusable>(null);
+
+  const onFocus = React.useCallback(() => {
+    setState({ focused: !state.focused });
+    if (buttonRef.current && !state.focused) {
+      const node = findNodeHandle(buttonRef.current);
+      console.log(node);
+      buttonRef.current.focus();
+    }
+  }, [state, setState]);
+  const testImage = require('./assets/icon_24x24.png');
```

Comment:
nit: could we move this to imports? I know it's test code..

Reply:
I'll just delete it, it's not really necessary.